internal/bminventory/inventory.go

```diff
@@ -2111,6 +2128,88 @@ func (b *bareMetalInventory) updateClusterData(_ context.Context, cluster *commo
 	return nil
 }
 
+func wereApiVipsUpdated(clusterVips []*models.APIVip, paramsVips []*models.APIVip) bool {
```

Comment:
Why not have generic function for both API and Ingress VIPs ?
Why is the type of APIVip different than IngressVip ?

Reply:
We got the two types generated from swagger.
I'll check for an option to unify to both checks to a single function.

Reply:
done